docs/customosbuildinstructions.md

```diff
@@ -49,8 +49,11 @@ Here are the expected contents of each subdirectory /file
    This is the [init script file](../kernelconfig/4.11/scripts/init_script)
 
 3. **/root** : this is the home directory of the root account. At this moment, it contains a sandbox file with a prebuilt empty ext4 fs for supporting Service VM operations
+         
          /root/integration/prebuildSandbox.vhdx
 
+         Note: this vhdx file could be found in opengcs/kernelconfig/4.11
```

Comment:
Oh I see. I think `Note: This is located under opengcs/kernelconfig/4.11` would be clearer.